### ISSUE TITLE
Update Readme with non default nats address

### ISSUE THREAD

**Issue by thogar-computer** [wasmCloud/wasmcloud-operator#33]
Hello, 
I have been following the instruction to deploy WASM within a Kubernetes cluster - pretty much got there now :) , however my pods for the operator are failing as they cannot connect to the NATS leafnode. 

seems the code defaults to ``nats.default``

https://github.com/wasmCloud/wasmcloud-operator/blob/35af8adb73238aa3d0834120f7cfeb1bdf771479/crates/types/src/v1alpha1/wasmcloud_host_config.rs#L120

however if a user is following the readme they set up the nats instance with a name of ``nats-cluster``

https://github.com/wasmCloud/wasmcloud-operator/blob/35af8adb73238aa3d0834120f7cfeb1bdf771479/README.md?plain=1#L105

This should be the change needed in the instructions to set the correct value

```yaml
  # The lattice to connect the hosts to
  lattice: 83a5b52e-17cf-4080-bac8-f844099f142e
  # Additional labels to apply to the host other than the defaults set in the operator
  hostLabels:
    some-label: value
  # The address to connect to nats 
  natsAddress: nats://nats-cluster.default.svc.cluster.local  
  # Which wasmCloud version to use
  version: 1.0.2
  # The name of a secret in the same namespace that provides the required secrets.
  secretName: my-wasmcloud-cluster
```

if noone has any issues, I am happy to open a PR

**Comment by joonas** [comment]
@thogar-computer hey, thank you for chasing that down, you're absolutely right that the values we have in the README would not work with the default in the operator. 

There's some historical context as to why we chose to set those up the way they are, and we really should change that, but in the meantime I would greatly appreciate it if you wouldn't mind opening a PR to update the instructions to match the correct values, that would be fantastic!

Thanks for digging into this!

**Comment by protochron** [comment]
Fixed by #34
